views/sdtdServer/playersTable.sejs

```diff
@@ -0,0 +1,204 @@
+    <%- await include('./partials/deletePlayersButton.sejs') %>
+    <%- await include('./partials/wipeTeleports.sejs') %>
+    <%- await include('./partials/wipeCurrencyBtn.sejs') %>
+    <div style="width: 275px" class="input-group float-right">
+      <div class="input-group-prepend">
+        <span class="input-group-text" id="basic-addon1"><i class="fa fa-search"></i></span>
+      </div>
+      <input id="search-input" type="text" class="form-control" placeholder="Search...">
+    </div>
+  </div>
+  <div class="card-body" style="padding: 0">
+    <div id="players-table" style="height: 100px; width: 100%;" class="ag-theme-alpine"></div>
+  </div>
+</div>
+
+<style>
+    .ag-root-wrapper {
+        border: 0!important;
+    }
+</style>
+
+<script>
+  let onlyOnline = null;
+  let height = null;
+  let lastEndRow = null;
+  let searchValue = '';
+  const columnDefs = [
+    { headerName: 'Name', field: 'name', width: 300, cellRenderer: (p) => {
+      if (!!p.value) {
+        return '<span>' + p.value + '</span>';
+      }
+    } },
+    { headerName: 'Entity ID', field: 'entityId' },
+    { headerName: 'Steam ID', field: 'steamId', width: 300 },
+    { headerName: 'Role', field: 'role', width: 150, cellRenderer: (p) => {
+      if (!!p.value) {
+        return p.value.name;
+      }
+    } },
+    { headerName: 'IP', field: 'ip', width: 200 },
+    { headerName: 'Country', field: 'country', width: 140, cellRenderer: (p) => {
+      if (!!p.value) {
+        let flagImg = '<img src="../../images/flags/blank.gif" class="flag flag-' + p.value.toLowerCase() + '"/>';
+        return flagImg + ' ' + p.value;
+      }
+    } },
+    { headerName: 'Playtime', field: 'playtime', cellRenderer: (p) => {
+      if (!!p.value) {
+        return (p.value / 3600).toFixed(1) + ' h';
+      }
+    } },
+    { headerName: 'Last online', field: 'lastOnline', cellRenderer: (p) => {
+      if (!!p.value) {
+        return new Date(p.value).toLocaleString();
+      }
+    } },
+    { headerName: 'Level', field: 'level', width: 130 },
+    { headerName: 'Currency', field: 'currency', width: 150 },
+    { headerName: 'Score', field: 'score', width: 130 },
+    { headerName: 'Deaths', field: 'deaths', width: 130 },
+    { headerName: 'Player Kills', field: 'playerKills', width: 170 },
+    { headerName: 'Zombie Kills', field: 'zombieKills', width: 170 },
+  ];
+
+  let gridOptions = {
+    columnDefs: columnDefs,
+    rowData: [],
+    defaultColDef: {
+      resizable: true,
+      sortable: true
+    },
+    enableSorting: true,
+    enableColResize: true,
+    enableCellTextSelection: true,
+    rowModelType: 'infinite',
+    cacheBlockSize: 25,
+    maxConcurrentDatasourceRequests: 1,
+    infiniteInitialRowCount: 50,
+    onGridReady: function (params) {
+      params.api.setDatasource(createDatasource());
+      params.api.sizeColumnsToFit();
+    },
+  };
+  $(document).ready(() => {
+    onlyOnline = <%= onlyOnline %>;
```

Comment:
could use `window.SAILS_LOCALS` here to keep it javascript-ey.

Reply:
That didnt worked for me, i tryed it before. it did not get passed to it